tox.ini

```diff
@@ -13,36 +13,32 @@ extras =
     test
 
 commands =
-    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} bootstrap
-    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} annotate
-    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} install test
-    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir}
+    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} buildout:sources-dir={toxinidir}/src bootstrap
+    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} buildout:sources-dir={toxinidir}/src annotate
+    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} buildout:sources-dir={toxinidir}/src install test
+    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} buildout:sources-dir={toxinidir}/src
     coverage run {envbindir}/test -v1 --auto-color {posargs}
```

Comment:
at the moment plone.testing and plone.app.testing are not python 3 compatible, 
@icemac has started to port them, but that is not finished or merged, so that will propably fail most of the time, now.

For the long run that is what we need.

Reply:
Thanks @loechel!
I am anyway using the test command to fix some packages (e.g. plone.supermodel, that do not require plone.testing).